tools/clang/test/DXC/strip_dbg.test

```diff
@@ -1,7 +1,7 @@
 // /Fd implies /Qstrip_debug ; path with / produces auto hash-named .pdb file
 // RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zi /Fd ./ /Fo %t.strip
 // RUN: %dxc -dumpbin %t.strip | FileCheck  %s --check-prefix=STRIP_DBG
-// RUN: file -E *.pdb && rm *.pdb
+// RUN: ls *.pdb && rm *.pdb
```

Comment:
I still feel like we should have changed the option above where it says `/Fd ./` to `/Fd %t.pdbdir/` after making that directory.  Then we don't have to remove the pdb here.